pom.xml

```diff
@@ -1030,7 +1030,7 @@
           <plugin>
             <groupId>org.apache.maven.plugins</groupId>
             <artifactId>maven-javadoc-plugin</artifactId>
-            <version>2.9.1</version>
+            <version>2.9.2-SNAPSHOT</version>
```

Comment:
I don't think we should bump this dependency. Was this a mistake?

Reply:
Reverted